Source/SecurityWrapper.swift

```diff
@@ -128,7 +128,7 @@ struct SecurityWrapper {
         
         let dict = [
             kSecClass as String         : kSecClassIdentity,
-            kSecReturnRef as String     : kCFBooleanTrue,
+            kSecReturnRef as String     : kCFBooleanTrue!,
```

Comment:
I think this is cleaner and just as effective:

```suggestion
            kSecReturnRef as String     : true,
```